src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -10,7 +10,7 @@ $font-family-icon: 'ForkAwesome';
 
 $badge-font-size: 1rem;
 $form-feedback-font-size: 100%;
-$popover-max-width: 350px;
+$popover-max-width: 700px;
```

Comment:
we shouldn't do this. If this is something that is needed only for multi-cluster the change it there only unless we know the impact of this change and clearly say that this won't break anything hidden that we don't know of yet